.github/workflows/main.yml

```diff
@@ -7,7 +7,28 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
     steps:
     - uses: actions/checkout@v2
-    - uses: jpetrucciani/black-check@19.10b0
+    - name: Build docker-compose
+      run: docker-compose up -d
```

Comment:
I'd add a `docker-compose build rmd` here to be sure it's actually being built.

Reply:
done